planning/obstacle_cruise_planner/include/obstacle_cruise_planner/node.hpp

```diff
@@ -193,6 +197,9 @@ class ObstacleCruisePlannerNode : public rclcpp::Node
     double lat_hysteresis_margin_for_slow_down;
     int successive_num_to_entry_slow_down_condition;
     int successive_num_to_exit_slow_down_condition;
+    // consideration for the current ego pose
+    bool enable_to_consider_current_pose;
+    double time_to_convergence;
```

Comment:
```suggestion
    bool enable_to_consider_current_pose{false};
    double time_to_convergence{1.5};
```
not 0.0. to prevent 0 divided
```
double rem_ratio = (time_to_convergence - time_elapsed) / time_to_convergence;
````